src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Endpoints/Completion.cs

```diff
@@ -322,6 +323,13 @@ private void AddSnippetCompletions(DelegatedCompletionParams request, ref Pooled
             return;
         }
 
+        // Don't add snippets if the trigger characters contain whitespace
+        if (request.Context.TriggerCharacter is not null
```

Comment:
Can't use  `string.IsNullOrWhitespace` because we do want to add for null and empty strings.